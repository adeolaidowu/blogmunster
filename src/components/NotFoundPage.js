import React from "react";
import { Link } from "react-router-dom";

const NotFoundPage = () => (
  <div>
    Error 404! Page Not Found - <Link to="/">Go Home</Link>
  </div>
);

export default NotFoundPage;
