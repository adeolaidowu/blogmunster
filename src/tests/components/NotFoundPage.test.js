import React from "react";
import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import NotFoundPage from "../../components/NotFoundPage";

test("should correctly render component", () => {
  const wrapper = shallow(<NotFoundPage />);
  expect(toJSON(wrapper)).toMatchSnapshot();
});
