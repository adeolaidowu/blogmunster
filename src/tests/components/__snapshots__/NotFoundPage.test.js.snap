// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render component 1`] = `
<div>
  Error 404! Page Not Found - 
  <Link
    to="/"
  >
    Go Home
  </Link>
</div>
`;
